UnityProject/Assets/Scripts/Health/Living/DNABloodTypeGen.cs

```diff
@@ -0,0 +1,50 @@
+using Random = UnityEngine.Random;
+
+[System.Serializable]
+public class DNAandBloodType
+{
+	public string DNAString;
+	public string BloodType;
+
+	enum bloodtypes { oneg, opos, bneg, bpos, aneg, apos, abneg, abpos}
+	
+	private int BloodTypeGenerator;
+	
+	public void BloodTypeandDNA()
+	{	
+		DNAString = System.Guid.NewGuid().ToString();
+		BloodTypeGenerator = Random.Range(1,1000);
+		if (BloodTypeGenerator <= 364)
+		{
+			BloodType = "opos";
```

Comment:
Change BloodType from string to type bloodtypes and assign bloodtypes.opos.

Also please change bloodtypes from all lower case to pascal case (i.e. BloodTypes) and change each type in the BloodTypes enum to camelcase (e.g. oNeg)